src/allmydata/web/retrieve-status.xhtml

```diff
@@ -1,43 +1,48 @@
-<html xmlns:n="http://nevow.com/ns/nevow/0.1">
+<html xmlns:t="http://twistedmatrix.com/ns/twisted.web.template/0.1">
+
   <head>
     <title>Tahoe-LAFS - Mutable File Retrieve Status</title>
     <link href="/tahoe.css" rel="stylesheet" type="text/css"/>
     <link href="/icon.png" rel="shortcut icon" />
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
   </head>
+
   <body>
 
-<h1>Mutable File Retrieve Status</h1>
-
-<ul>
-  <li>Started: <span n:render="started"/></li>
-  <li>Storage Index: <span n:render="si"/></li>
-  <li>Helper?: <span n:render="helper"/></li>
-  <li>Current Size: <span n:render="current_size"/></li>
-  <li>Progress: <span n:render="progress"/></li>
-  <li>Status: <span n:render="status"/></li>
-</ul>
-
-<h2>Retrieve Results</h2>
-<ul>
-  <li n:render="encoding" />
-  <li n:render="problems" />
-  <li>Timings:</li>
-  <ul>
-    <li>Total: <span n:render="time" n:data="time_total" />
-    (<span n:render="rate" n:data="rate_total" />)</li>
+    <h1>Mutable File Retrieve Status</h1>
+
+    <ul>
+      <li>Started: <span t:render="started"/></li>
+      <li>Storage Index: <span t:render="si"/></li>
+      <li>Helper?: <span t:render="helper"/></li>
+      <li>Current Size: <span t:render="current_size"/></li>
+      <li>Progress: <span t:render="progress"/></li>
+      <li>Status: <span t:render="status"/></li>
+    </ul>
+
+    <h2>Retrieve Results</h2>
+
     <ul>
-      <li>Fetching: <span n:render="time" n:data="time_fetch" />
-      (<span n:render="rate" n:data="rate_fetch" />)</li>
-      <li>Decoding: <span n:render="time" n:data="time_decode" />
-      (<span n:render="rate" n:data="rate_decode" />)</li>
-      <li>Decrypting: <span n:render="time" n:data="time_decrypt" />
-      (<span n:render="rate" n:data="rate_decrypt" />)</li>
+      <li t:render="encoding" />
```

Comment:
No action required, but re-indenting this does make it a little harder to review.